app/screens/Lists/BooksList/components/BookItem/index.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import { View, Image, StyleSheet, Text } from 'react-native';
+
+function BookItem({ image, title, author }) {
+  return (
+    <View style={styles.itemContainer}>
+      {image ? (
+        <Image
+          style={styles.image}
+          source={{ uri: image }}
+        />
+      ) : (
+        <View style={styles.placeholderImage} />
+      )}
+      <View style={styles.textContainer}>
+        <Text style={styles.itemTitle}>{title}</Text>
+        <Text style={styles.itemAuthor}>{author}</Text>
+      </View>
+    </View>
+  );
+};
+
+BookItem.defaultProps = {
+  image_url: ''
```

Comment:
Mmm maybe you can find a placeholder image to the component and use it like default if there is not image_url

Reply:
this props is called `image` not `image_url`